### PR TITLE
Polish localization

### PR DESCRIPTION
**Pull request by TacoConKvass**

### CODE REVIEW

File: Localization/pl-PL.hjson [hunk 1]
```diff
@@ -2,154 +2,154 @@ Mods: {
...
-					// Tooltip: Change to select which list will be used for the item filter
+					Label: Zmiana wyboru listy w użyciu
+					Tooltip: Zmień, żeby wybrać która lista ma być używana do filtrowania przedmiotów.
 				}
 
 				Buff: {
-					// Label: Buff
-					// Tooltip: Toggle if having the magnet active gives you a buff
+					Label: Wzmocnienie
+					Tooltip: Przełącz, żeby włączenie magnesu dawało graczowi incormujące o tym wzmocnienie.
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: typo? should be informujące

File: Localization/pl-PL.hjson [hunk 2]
```diff
@@ -2,154 +2,154 @@ Mods: {
...
 				}
 			}
 
 			FilterListModeType: {
-				// Tooltip: ""
-				// Blacklist.Label: Blacklist
-				// Whitelist.Label: Whitelist
+				Tooltip: ""
+				Blacklist.Label: Biała lista
+				Whitelist.Label: Czarna lista
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: These are swapped

**Comment by TacoConKvass** [comment]
Will fix in the evening

**Comment by TacoConKvass** [comment]
Fixed, that's what I get for *doing translations past midnight XD

File: Localization/pl-PL.hjson [hunk 3]
```diff
@@ -2,154 +2,154 @@ Mods: {
...
+				Tooltip: ""
+				Bosses.Label: Zwiększaj wraz z pokonanymi bossami
+				AlwaysMaxRange.Label: Bossowie + Maksymalny zasięg
+				OnlyConfig.Label: Własne + Maksymalny zasięg
 			}
 
-			// Common.AcceptClientChanges: Only the host of this world can change the config! Do so in singleplayer.
-		}
+			Common.AcceptClientChanges: Tylko gospodarz tego świata może zmieniać ustawienia konfiguracji! Zrób to w trybie jednoosobowym.
+		}
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: unnecessary space

File: Localization/pl-PL.hjson [hunk 4]
```diff
@@ -2,154 +2,154 @@ Mods: {
...
+					Tooltip: Przełącz, jeśli magnes ma przyciągać monety.
 				}
 
 				PickupEffect: {
-					// Label: "[i:FragmentNebula] Pickup Effect Items"
-					// Tooltip: Toggle if items like nebula armor boosters or modded items like the music notes from Thorium should be picked up by the magnet
-				}
+					Label: "[i:FragmentNebula] Przedmioty z efektem przy podnoszeniu"
+					Tooltip: Przełącz, jeśli magnes ma przyciągać rzeczy takie jak wzmocnienia pochodzące ze Zbroi Nebuli, czy też rzeczy pochodzące z modyfikacji takie jak nuty z Thorium.
+                                }
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: whitespace change. should be tabs, not spaces

Comment (TacoConKvass) [reply Localization/pl-PL.hjson]: Fixed

File: Localization/pl-PL.hjson (hunk 3 above)
Comment (TacoConKvass) [reply Localization/pl-PL.hjson]: Not sure what do you mean, I cant see anything here

Comment (TacoConKvass) [reply Localization/pl-PL.hjson]: Done

**Pull request merged**